examples/article-access-laterpay.amp.html

```diff
@@ -233,15 +236,8 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section id="meter-notif" amp-access="views > 0" amp-access-hide class="meter-section">
-          <template amp-access-template type="amp-mustache">
-            You are viewing article {{views}} of {{maxViews}}!
-          </template>
-          <a on="tap:meter-notif.hide">[Close]</a>
-        </section>
-
         <section amp-access="NOT error AND NOT access" amp-access-hide class="login-section">
-          <a on="tap:amp-access.login">Login to read more!</a>
+          <amp-access-laterpay-list></amp-access-laterpay-list>
```

Comment:
Ok. Let's start with `id="amp-access-laterpay-dialog"` format. And we'll adjust as needed.